src/LinearOperators.jl

```diff
@@ -16,11 +16,15 @@ type LinearOperatorException <: Exception
 end
 
 # when indexing, Colon() is treated separately
-typealias LinearOperatorIndexType Union{UnitRange{Int}, StepRange{Int, Int}, AbstractVector{Int}}
+const LinearOperatorIndexType = Union{UnitRange{Int}, StepRange{Int, Int}, AbstractVector{Int}}
```

Comment:
It's not untested, it appears on `opRestriction` that is tested on `test_linop.jl`.